tests/basic/upgrade_test.go

```diff
@@ -19,7 +22,15 @@ import (
 var storkLabel = map[string]string{"name": "stork"}
 
 const (
-	pxctlCDListCmd = "pxctl cd list"
+	pxctlCDListCmd     = "pxctl cd list"
+	ibmHelmRepoName    = "ibm-helm-portworx"
+	ibmHelmRepoURL     = "https://raw.githubusercontent.com/portworx/ibm-helm/master/repo/stable"
```

Comment:
just checking if we can pass the RepoURL from config maps instead of hardcoding ?